### ISSUE TITLE
Add an option to ultimately enable plain text copy/paste

### ISSUE THREAD

**Issue by aurora-of-earth** [hluk/CopyQ#1147]
The contrast of a styled HTML text snippet when added to a plain text item is undesirable.

Better to have an option to disable all HTML rendering.

**Comment by hluk** [comment]
This is mentioned in FAQ: https://copyq.readthedocs.io/en/latest/faq.html#how-to-paste-as-plain-text

**Comment by aurora-of-earth** [comment]
@hluk 
This is so key a feature to be available by default.

**Comment by hluk** [comment]
Did you try following?

> You can also disallow rich text storing: go to preferences, “Items” tab, uncheck “Web” checkbox and under “Text” uncheck “HTML” checkbox.

**Comment by hluk** [comment]
I've updated the documentation to make it more clear: https://copyq.readthedocs.io/en/latest/faq.html#how-to-store-only-plain-text

**Comment by aurora-of-earth** [comment]
@hluk yes, but it's still storing in HTML format.
There's no 'Web' item in Preferences > Items.

**Comment by hluk** [comment]
> @hluk yes, but it's still storing in HTML format.
> There's no 'Web' item in Preferences > Items.

You probably just don't have the Web plugin installed.

So if you copy something after disabling the HTML storing the new item still shows HTML format? (You can check this by selecting an item and press F4 - text/html shouldn't be in the format list.)

**Comment by aurora-of-earth** [comment]
@hluk I don't have the web plugin installed, should I?
When I press F4 on an item it still shows the html format in the options list of the text copied was in HTML format.
I want it to ultimately store everyhting in plain text regardless of the input format.

**Comment by hluk** [comment]
> @hluk I don't have the web plugin installed, should I?

No, I don't think it's available on all platforms and it's not even worth installing (I'm actually trying to slowly deprecate it).

> When I press F4 on an item it still shows the html format in the options list of the text copied was in HTML format.
> I want it to ultimately store everyhting in plain text regardless of the input format.

Did you disable storing HTML? It's in configuration Items tab.

![image](https://user-images.githubusercontent.com/38054/62290915-c1a32500-b462-11e9-83fc-1ffcfe4b5a11.png)

This will work only for newly copied items. It won't affect existing items but you can delete "text/html" from the F4 dialog.

If this doesn't work, you may have some commands (in command dialog - F6) that require "text/html" input format.

**Comment by aurora-of-earth** [comment]
> Did you disable storing HTML? It's in configuration Items tab.

Yes.
The real issue is that I wanna permanently disbale all styling for the text snippets.
